docs/install-macos.md

```diff
@@ -0,0 +1,86 @@
+Open Terminal.app by clicking the magnifying glass icon in the top right of your screen.
```

Comment:
Nitpicking: could we have a first line that says e.g. `# macOS Install guide`?

Reply:
and maybe specify what version of macOS this guide was written for?

Reply:
(it all looks like stuff that isn't going to change any time soon, but it could help with the inevitable "I tried this on my old 10.11 machine and it didn't work")

Reply:
maybe e.g. a  `## requirements / intro` and a `## installing`

Reply:
What would you expect to see in the intro other than explaining the guide was written with macOS 11.1?

Locally I've tried out a note:

![](https://p198.p4.n0.cdn.getcloudapp.com/items/ApuG52m5/715a209b-c363-40ae-903f-bd4dafbe9243.jpg?v=48738beae917b051d60a25060a9b868d)

I think it needs a bit of body on that note to set expectations but otherwise I'm not sure what else to put in there!

Reply:
For me that's enough - as it gives me a reference point to a working setup - suggestions for things you could add include ", but should work with 10.14 or higher", or in future " and tested with macOS 12, 13, ...". 

(a quick scout suggests that homebrew & docker ask for 10.14, GHdesk & vscode only want 10.10)